src/go/k8s/pkg/console/deployment_internal_test.go

```diff
@@ -0,0 +1,116 @@
+package console
```

Comment:
nit: why not deployment_test? Why internal? It does not map to any convention I know of that we would be following

Reply:
this is to skip golangci-lint testpackage which requires it to be in `console_test` package, we are testing internal functions here

Reply:
Couldn't you add // nolint comment here and rename the file?